hexrd/ui/save_images_dialog.py

```diff
@@ -49,9 +49,9 @@ def save_images(self):
         for det in dets:
             filename = f'{self.ui.file_stem.text()}_{det}.{selected_format}'
             path = f'{self.parent_dir}/{filename}'
-            if selected_format.startswith('HDF5'):
+            if selected_format.startswith('hdf5'):
```

Comment:
Is `selected_format` always lowercase?

Reply:
Yes, it is just the text from the drop-down menu, set [here](https://github.com/HEXRD/hexrdgui/blob/8c26f189253165b7ec263935ed55e849db0ff20c/hexrd/ui/save_images_dialog.py#L45). I will add in a guard for that though.